hooks/command

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+export DOCKER_CLI_EXPERIMENTAL=enabled
+export AWS_PAGER=""
+
+build="${BUILDKITE_BUILD_NUMBER:-0}"
+repository="$BUILDKITE_PLUGIN_DYNAMODB_IMAGE_REPOSITORY"
+tables=()
+script_dir="$(dirname "${BASH_SOURCE[0]}")"
+local_dynamo_container_id=""
+local_dynamo_port=8000
+tmp_dir="${script_dir}/tmp"
+mkdir -p $tmp_dir
```

Comment:
I'd also be keen on running ShellCheck on this script, as it should enforce some conventions around escaping/quoting:

```suggestion
mkdir -p "${tmp_dir}"
```

Reply:
Will run ShellCheck on it 👍